### PR TITLE
Include stats calculator to ease performance evaluation

### PR DESCRIPTION
**Pull request by MatteoDrago**
First need to merge #4

### CODE REVIEW

File: examples/vr-app-n-stas.cc [hunk 1]
```diff
@@ -168,6 +169,10 @@ main (int argc, char *argv[])
   YansWifiChannelHelper channel = YansWifiChannelHelper::Default ();
   YansWifiPhyHelper phy;
   phy.SetChannel (channel.Create ());
+  phy.Set("ChannelSettings",
+          StringValue(std::string("{0, ") + std::to_string(channelWidth) +
+                      ", " + (frequency == 2.4 ? "BAND_2_4GHZ" : "BAND_5GHZ") +
+                      ", 0}"));
```
Comment (mattia-lecci) [inline examples/vr-app-n-stas.cc:175]: a similar comment was done for PR #4

**Review by mattia-lecci — COMMENTED** [review]
i think this PR might conflict with PR #4 , right? i assume #4 should be accepted first and #5 should only include the relevant changes

**Comment by MatteoDrago** [comment]
> 

Yes, I wrote it in the PR description 👍🏻

**Comment by mattia-lecci** [comment]
this PR now conflicts with the main, could you please solve them?

**Comment by MatteoDrago** [comment]
rebased and solved conflicts, should now be ok

**Review by mattia-lecci — APPROVED** [review]

**Pull request merged**